openlibrary/templates/support.html

```diff
@@ -9,29 +9,28 @@ <h1>$_("How can we help?")</h1>
 <div id="contentBody">
 
  $if done:
-   <em>Your question has been sent to our support team. We'll get back to you shortly. You can also <a href="https://twitter.com/openlibrary">contact us on Twitter</a>.</em>
+   <em>$:_('Your question has been sent to our support team. We\'ll get back to you shortly. You can also <a href="https://twitter.com/openlibrary">contact us on Twitter</a>.')</em>
```

Comment:
Mixed on this one; most of our staff will only be able to respond to English emails; so translating this page could confuse. Can you put the changes to this file in separate PR? That'll let use merge your other changes faster while we get more thoughts on this :)

Reply:
New PR created (#4754).